src/store/reducers/overview/overview.ts

```diff
@@ -1,18 +1,28 @@
+import type {Timeout} from '../../../types/api/query';
 import {api} from '../api';
 
 export const overviewApi = api.injectEndpoints({
     endpoints: (build) => ({
         getOverview: build.query({
-            queryFn: async ({paths, database}: {paths: string[]; database: string}, {signal}) => {
+            queryFn: async (
+                {
+                    paths,
+                    database,
+                    timeout,
```

Comment:
I think you should exclude `timeout` (and probably `concurrentId`) in `serializeQueryArgs`, otherwise your `getTableSchemaData` will always initiate a new request and will never reuse data from other request to `overviewApi`

Reply:
In current versions there is always a new request, even if entity overview have already been loaded

Reply:
removed timeout and concurrentId
now queries use same api and timeout moved to front with Promise.race and wait